pkg/template/generator/expressionvalue.go

```diff
@@ -94,6 +95,8 @@ func replaceWithGenerated(s *string, expression string, ranges [][]byte, length
 			alphabet += Numerals
 		case `\a`:
 			alphabet += Alphabet + Numerals
```

Comment:
barring the ability to change that at this point, @liggitt is probably right, we should just add each "group" to the pool of potential characters once.